src/control/server/ctl_ranks_rpc.go

```diff
@@ -167,10 +168,11 @@ func (svc *ControlService) StopRanks(ctx context.Context, req *ctlpb.RanksReq) (
 		return nil, errors.New("no ranks specified in request")
 	}
 
-	signal := syscall.SIGINT
-	if req.Force {
-		signal = syscall.SIGKILL
-	}
```

Comment:
Why make all of these changes? You are (or someone else is) just going to have to change everything back later. This could have been a one-line change, maybe with some extra comments. What you could do is define a const, e.g. DefaultStopSignal, and then when things change back you only need to change it in one place.

Reply:
Looks like this is the version @gnailzenh is in favour of where prep_shutdown/disable_exclude behaviour is preserved for the non-force and no-ranks-specified `dmg system stop` controlled shutdown case.

Reply:
@mjmac can we go with this one as an urgent fix please?